src/themes/default/stylesheets/main.scss

```diff
@@ -67,7 +68,7 @@ $background-arguments: #fafbfc;
 
 @import 'custom';
 @import 'base';
-@import 'monokai';
+@import 'syntax-highlighting';
```

Comment:
This could just be named `syntax`. @newhouse wdyt?

Reply:
Good by me!